tests/test_web.py

```diff
@@ -0,0 +1,19 @@
+""" Tests for web """
+
+import pytest
+from ubiquerg.web import is_url
+
+
+@pytest.mark.parametrize("string", ["https://www.github.com",
```

Comment:
Not a big deal, but I try to avoid variable names that are also object names since sometimes it can lead to unexpected behavior that can be weird to diagnose. `string` used to be a module and I think still is so I may just pick something like `"s"` for this, but again not a big deal if you want to leave it.